dev-cluster/deploy.sh

```diff
@@ -3,13 +3,22 @@ set -e
 
 cd "${0:A:h}"
 
+if [[ $(uname) == "Darwin" ]]; then
+  export K3D_FIX_DNS=0
+  if command -v colima >/dev/null 2>&1; then
+    if colima status >/dev/null 2>&1; then
```

Comment:
```suggestion
  if colima status &> /dev/null; then
```

It's not like running a program that doesn't exist could ever succeed, could it?

Reply:
Vibes :P 

But true, will change.